### DISCUSSION TITLE
Should buildtime dependencies go in devDependencies?

### DISCUSSION THREAD

**Discussion by hahnbeelee** [vercel/next.js#40858] [Help]
I saw discussion [#14000](https://github.com/vercel/next.js/discussions/14000) that says "it doesn't matter if you put dependencies in dev or not". But then I also came across [#4496](https://github.com/vercel/next.js/issues/4496) that discusses decreasing cold start times. Decreasing cold start times is something really important to me and I just want to know what I should be doing with my dependencies. 

For simplicity of my question, should the dependencies I use solely during build time/in `next.config.js` be moved to `devDependencies`?

**Comment by sirnovikov** [comment]
Looks like build-time dependencies should go to devDependencies.
> The convention is to list runtime dependencies under dependencies and build-time dependencies under devDependencies.

https://vercel.com/guides/dependencies-from-package-json-missing-after-install
